tests/integration/test_payment.py

```diff
@@ -0,0 +1,184 @@
+import os
+import string
+import time
+
+import pytest
+
+from jumpscale.loader import j
+
+WALLET_NAME = os.environ.get("WALLET_NAME")
```

Comment:
who should set this?

Reply:
Anyone who wants to run the test, and it will raise an error if it's not set.